pinecone/sparse/splade/splade-quora.ipynb

```diff
@@ -373,10 +373,7 @@
       "source": [
         "import pinecone\n",
         "\n",
-        "pinecone.init(\n",
```

Comment:
The example query in this notebook is not really interesting.  
The sparse results are completely irrelevant, and the hybrid results look just like the dense ones.  

We should find a better example where we can truly show the benefit of hybrid over both dense and sparse

Reply:
@jamescalam I actually run a script to find a question from the questions corpus, but couldn't find a question that really require hybrid to boost performance, any ideas on that?

Reply:
In my next iteration I plan to replace it with the home depot dataset for which I already found intreating examples for hybrid search